server/testsupport/inductionResponseTestDataBuilder.ts

```diff
@@ -1,6 +1,170 @@
 import type { InductionResponse } from 'educationAndWorkPlanApiClient'
 
-const aValidInductionResponse = (options?: {
+const aLongQuestionSetInduction = (
```

Comment:
This test data builder is now equivalent to `ciagInductionTestDataBuilder` in that it exports a Long Question Set Induction and a Short Question Set Induction.
The fields that it populates for both is semantically equivalent to the CIAG Induction equivalents.